fitlins/cli/run.py

```diff
@@ -87,22 +88,30 @@ def get_parser():
     g_bids.add_argument('--space', action='store',
                         choices=['MNI152NLin2009cAsym', ''],
                         default='MNI152NLin2009cAsym',
-                        help='registered space of input datasets. Empty value for no explicit space.')
+                        help='registered space of input datasets. '
+                             'Empty value for no explicit space.')
     g_bids.add_argument('--force-index', action='store', default=None,
                         help='regex pattern or string to include files')
     g_bids.add_argument('--ignore', action='store', default=None,
                         help='regex pattern or string to ignore files')
     g_bids.add_argument('--desc-label', action='store', default='preproc',
                         help="use BOLD files with the provided description label")
+    g_bids.add_argument('--database-path', action='store', default=None,
+                        help="Caution, this is an Expert level option subject to change! "
```

Comment:
Not sure if this caution is necessary given how few people use this, and how it's overall an early stage project.